src/guided_fuzzing_daemon/nyx.py

```diff
@@ -46,6 +46,9 @@ def nyx_main(
 
     config_file = opts.sharedir / "config.sh"
 
+    log_tee = LogTee(opts.afl_hide_logs, opts.instances)
+    tmp_base = Path(mkdtemp(prefix="gfd-"))
```

Comment:
This directory is not cleaned up in all cases.

Reply:
Fixed in [6f2d217](https://github.com/MozillaSecurity/guided-fuzzing-daemon/pull/62/commits/6f2d217805f3529489a15dc33364d84e94bdccd8).